src/kqueue.rs

```diff
@@ -371,6 +371,11 @@ impl EventExtra {
     pub fn is_pri(&self) -> bool {
         false
     }
+
+    #[inline]
+    pub fn is_connect_failed(&self) -> bool {
+        unimplemented!("is connect failed is not supported on kqueue");
```

Comment:
Just return `false` here, no need to panic

Reply:
updated